libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -328,7 +300,10 @@ static int CALLBACK loadDialogBrowseCallback(
     std::string defaultPath = *(std::string*)lpData;
     if(defaultPath!="" && uMsg==BFFM_INITIALIZED){
 		wchar_t         wideCharacterBuffer[MAX_PATH];
-		wcscpy(wideCharacterBuffer, convertNarrowToWide(ofToDataPath(defaultPath)).c_str());
+		
+		
+		std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+		wcscpy(wideCharacterBuffer, converter.from_bytes(ofToDataPath(defaultPath)).c_str());
```

Comment:
out of my comfort zone here but I wonder if the native handling does not simplify things as `defaultPath` is now an `filesystem::path`? right now it looks as it's converted to a std::string then converted back to wide buffer.